Medals/PersonalBestMedal.as

```diff
@@ -0,0 +1,108 @@
+class PersonalBestMedal : Medal
+{
+	bool IsVisible() override { return showPbest; }
+	protected string GetName() override { return pbestText; }
+	protected int GetScoreAsync(CGameCtnChallenge@ map) override
+	{
+		int score = -1;
```

Comment:
Since the MP4 current implementation can yield, shouldn't m_cachedScore also be cleared somewhere here, at least for MP4? (Unless I'm missing where that happens because I'm using my phone to read a code diff)

Except if you come up with a better implementation that doesn't yield, of course.

Reply:
True, we should probably clean that just before assigning it. I still want to look at the yielding on Turbo & Maniaplanet though, since I don't think it needs to be this complicated 🤔